src/Tizen.MachineLearning.Train/Interop/Interop.Libraries.cs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+internal static partial class Interop
+{
+    internal static partial class Libraries
+    {
+        public const string Nntrainer = "libcapi-nntrainer.so";
```

Comment:
how about using cc-api to develop C# API.

Reply:
TizenFX uses DllImport to eliminate library dependencies.
However, as a technical issue, there is a problem of symbol name mangling in C++.
So you have to use Tizen native capi.